modules/govuk/manifests/procfile/worker.pp

```diff
@@ -66,6 +67,10 @@
         service_description => "${title} procfile worker upstart up",
         host_name           => $::fqdn,
       }
+      collectd::plugin::process { "app-worker-${title_underscore}":
+        ensure => $ensure,
+        regex  => "sidekiq .* ${title}.*\\.gov\\.uk",
+      }
```

Comment:
If you put this block outside the `if $ensure == present {`, the use of `$ensure` in the block will mean that the `collectd::plugin::process` thing is removed if `$ensure` specifies it should be. At the moment, `$ensure` will always be `present` here, as that is the condition in the `if`.